packages/dart/noports_core/lib/src/commands/utils/regex.dart

```diff
@@ -0,0 +1,21 @@
+/// Regular expressions for parsing noports command arguments.
+class ActivateRegex {
+  // CRAM authentication: <atsign>:cram:<secret>
+  static final cram = RegExp(r'^(?<atsign>[^:]+):cram:(?<secret>.+)$');
+
+  // Enrollment: <atsign>:enroll:otp:<otp>[:name:<device>:keyfile:<path>]
+  static final enroll = RegExp(
+    r'^(?<atsign>[^:]+):enroll:otp:(?<otp>[A-Za-z0-9]{6})'
```

Comment:
This is ordered, which is fine simple is best. Just make sure the Desktop app respects the same order  when it gets implemented there.

Ideally the app consumes the same classes.